pandas/core/frame.py

```diff
@@ -831,7 +835,31 @@ def _can_fast_transpose(self) -> bool:
         if len(blocks) != 1:
             return False
 
-        return not self._mgr.any_extension_types
+        dtype = blocks[0].dtype
+        # TODO(EA2D) special case would be unnecessary with 2D EAs
+        return not is_1d_only_ea_dtype(dtype)
+
+    @property
```

Comment:
can you just incorporate in `.values`?

Reply:
.values is public so thatd be an API change.  id like for ._values to eventually return a 2D EA in these cases, but didnt think this was the right place to push for that.  ill try making that change and see what happens

Reply:
tried changing ._values and to my surprise only one test broke.  ill keep poking at it and we can revisit after #40456

Reply:
i have a branch now that succesffully does this, saving it as a follow-up